libraries/core-react/src/Scrim/Scrim.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { forwardRef, useEffect, MouseEvent } from 'react'
+import styled from 'styled-components'
+import { scrim as tokens } from './Scrim.tokens'
+
+const { height, width, background } = tokens
+
+type StyledScrimProps = {
+  onClick: (event: MouseEvent, open: boolean) => void
+  isDismissable?: boolean
+}
```

Comment:
Don't think this duplicate type declarations is needed as you have defined the same in `Props`

Reply:
Hmm, yes good point